### ISSUE TITLE
WCAG 2.1: Reflow - Shortcut text information in settings is truncated at 200% zoom 

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#766]
**Describe the bug**
Text information for ```shift+F5``` shortcut is truncated in Settings -> Application

**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1.  Navigate to Settings -> Application in AI-Windows
2. Observe that the complete text information for ```Shift+F5``` is not visible 

**Expected behavior**
Text content should not be hidden and be made available without requiring to scroll in two dimensions

**Actual behavior**
Text content  is hidden

**Screenshots or .GIF**
![2](https://user-images.githubusercontent.com/22068434/79995369-009f5d80-846c-11ea-8416-23e0b4559115.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.
